pagopa/io-dev-api-server
# [IOCOM-577] Removal of the `category` property from `PaymentProblemJson`

## Short description
This PR removes the `category` property from `PaymentProblemJson`.

## List of changes proposed in this pull request
- updated definitions
- removed the category property from the generated payload

## How to test
Generate a payment that contains an error or that is already paid and check that the verify endpoint returns a payload without category.
